frontend/src/app/main/data/workspace/texts.cljs

```diff
@@ -289,14 +289,6 @@
 
 ;; --- RESIZE UTILS
 
-(defn update-overflow-text [id value]
-  (ptk/reify ::update-overflow-text
-    ptk/UpdateEvent
-    (update [_ state]
-      (let [page-id (:current-page-id state)]
-        (update-in state [:workspace-data :pages-index page-id :objects id] assoc :overflow-text value)))))
-
-
```

Comment:
Why is this removed? Perhaps because text overflow cannot occur now?

Reply:
Yes, the text overflowed is now displayed. This is something that I changed after talking to the designers.